psi4/driver/procrouting/sapt/sapt_mp2_terms.py

```diff
@@ -239,6 +239,10 @@ def df_mp2_fisapt_dispersion(wfn, primary, auxiliary, cache, do_print=True):
     ret = {}
     ret["Exch-Disp20,u"] = scalars["Exch-Disp20"]
     ret["Disp20,u"] = scalars["Disp20"]
+
+    fisapt.sinf_disp(matrix_cache, vector_cache, False)
```

Comment:
Can you add an option to optionally compute this in `psi4/src/read_options.cc:846` under with something like `options.add_bool("DO_DISP_EXCH_SINF", false);` similar to the IND version with an appropriate `if` statement here.